tests/unit/igraph_i_umap_fit_ab.c

```diff
@@ -31,10 +31,10 @@
 
 int main () {
     int i;
```

Comment:
Can you make this a `size_t` instead of `int`?

There are still some `int`s in tests/examples that should really be `igraph_integer_t`. Here it's not a problem to have an `int` but the fewer we have the least distraction there will be when looking for inappropriate uses. `size_t` is appropriate for indexing into raw arrays, like here, and won't cause noise when `grep`ping for `int`s.